src/server/string_family.cc

```diff
@@ -656,6 +664,10 @@ void SetCmd::RecordJournal(const SetParams& params, string_view key, string_view
     cmds.push_back("KEEPTTL");
   }
 
+  if (params.flags & SET_STICK) {
+    cmds.push_back("STICK");
```

Comment:
why did you decide adding it?

Reply:
Because there is no way to set a sticky value in a non-atomic script. I noticed this when I made an example and thought its kinda silly

Reply:
you know we do not support it in full sync/rdb snapshot, right?

Reply:
Doesn't matter for replication because only master evicts item, snapshotting might be an issue

Reply:
It matter for replication as well because during failover the notions of sticky items should be kept by slave